sechub-doc/src/docs/asciidoc/documents/pds/product_delegation_server_config_example1.json

```
@@ -25,7 +25,10 @@
                     {
                         "key" : "product1.add.tipoftheday",
                         "description" : "boolean as string, when 'true' we add always a tip of the day as info to results"
-                    }  
+                    }, {
+                        "key" : "pds.config.supported.datatype", //<11>
```

This should be plural form:
```suggestion
                        "key" : "pds.config.supported.datatypes", //<11>
```